packages/react-frontend/src/NewComment.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import { postComment } from "./apis";
+import "./NewComment.css";
+
+function NewComment(props) {
+  const [comment, setComment] = useState({
+    body: "",
+  });
+
+  function handleChange(event) {
+    setComment((values) => ({
+      ...comment,
+      [event.target.id]: event.target.value,
+    }));
+  }
+
+  function handleSubmit(event) {
+    postComment(
+      {
+        body: comment.body,
+        author: "Matt", //just set to prefix name until it works with oauth
```

Comment:
Will be able to do `req.session.name` after my changes.

Reply:
Sick